test/client-test.js

```diff
@@ -442,13 +442,16 @@ describe('SOAP Client', function() {
         var data = {
           Request: {
             $attributes: {
-              $xsiType: '{xmlnsTy}Ty'
+              $xsiType: {
```

Comment:
It was not supported by `soap` module but I added it. If it's not working, we need to fix it.